### ISSUE TITLE
Clarification request: INSPIRE themes for groundwater extraction

### ISSUE THREAD

**Issue by HanWelmer** [INSPIRE-MIF/helpdesk#39]
In the Netherlands the use of groundwater is to a large extent subject to licensing. As part of a license a requirement may be imposed to periodically report on the volume of extracted groundwater. Information on these matters has to be submitted for registration in the Key Registry on the Subsurface [NL: Basisregistratie Ondergrond, or BRO in short). Relevant datasets within the BRO are Groundwater Usage Facilities (GUF) and Groundwater Production Dossier (GPD].
 
As part of the standardization activities we will create mappings to relevant INSPIRE themes. At first we considered EF (Environmental Monitoring Facilities) and PF (Production and Industrial Facilities). But comparing the scope of these themes with those of the BRO we have second thoughts.
 
The scope of GUF and GPD is:  Within the scope of the BRO domain [consisting of groundwater usage facilities and groundwater production dossiers] fall all types of groundwater usage that are subject to a license or reporting requirement within the scope of the Environment Law – groundwater extraction (irrespective of purpose or quantity) and groundwater based subsurface energy systems--. This includes groundwater extraction for purposes such as drinking water, fire extinguishing water, livestock drinking water, irrigation, process water, cooling water, drainage by well points, soil or groundwater decontamination or level controlled land drainage  - using a pump-. [NL: Binnen de scope van dit BRO domein vallen alle vergunnings- of meldingsplichtige vormen van grondwatergebruik die vallen onder de Omgevingswet--onttrekkingen (ongeacht gebruiksdoel en of grootte) en bodemenergiesystemen--.  Dit betreft  o.a. onttrekkingen ten behoeve van drinkwater, bluswater, veedrenking, beregening, proceswater, koelwater, bronbemaling,  bodem en of grondwatersanering en peilgestuurde drainage-met pomp-].
 
For the sake of comparison I here quote the EF and PF scope from their respective data specifications:
•	EF: […] the focus is on environmental monitoring facilities as a linking element between spatial data themes as defined by the INSPIRE Directive and observations and measurements on specific aspects of the environment (e.g. air quality, atmospheric conditions, water quality).
•	PF: features related to production and industry, as well as entities related to describing summary information about the activities taking place in production and industrial facilities, and the main environmental issues related to them (pollution prevention, waste management, risk). [...] Extraction of resources includes the following: non-energy extractive industry (mining of construction materials, industrial minerals, and metallic minerals), energy extractive industry, and water.
 
Therefore the following questions:
1.	Should ground water extraction facilities for the purpose of drinking water be mapped to EF, or PF or neither?
2.	Should ground water extraction facilities for other purpose as listed above for GUF and GPD be mapped to EF, or PF or neither?
3.	Should production figures of extracted ground water for the purpose of drinking water be mapped to EF, or PF or neither?
4.	Should production figures of extracted ground water for other purposes be mapped to EF, or PF or neither?
5.	Are groundwater based subsurface energy systems within or outside the scope of EF and PF and also ER: Energy Resources?

Thank you very much in advance.

**Comment by KathiSchleidt** [comment]
Seems similar to water quantity in rivers that we've been modelling as EF.

Advantage of the EF model is that it allows for dynamic data, extraction over time, through the integrated O&M model. If you go this direction, I'd recommend investigating the use of OGC SensorThings API (STA), now accepted as an [INSPIRE Good Practice](https://inspire.ec.europa.eu/good-practice/ogc-sensorthings-api-inspire-download-service), demonstrators (including water surface quantity) available from the [API4INSPIRE pages](https://datacoveeu.github.io/API4INSPIRE/datanests/franco-germanic-flow.html)

**Comment by HanWelmer** [comment]
I understand... groundwater extraction can be compared with water extraction from a river.

I also found that drinking water extraction points are mapped to EF as prioritaire dataset nr. 21.02. See https://github.com/INSPIRE-MIF/need-driven-data-prioritisation/blob/main/documents/eReporting_PriorityDataList_V2.1_final_20201008.xlsx

Next the relation of this to my questions 1 and 3: map facilities and production figures to EF? Or map the facilities also to PF?

Secondly, in the Netherlands we make a distinction between groundwater extraction for the purpose of drinking water and groundwater extraction for other purposes. That explains my questions 2 and 4.

Question 5 is of course related to the use of groundwater, but not for any kind of consumption (extraction + infiltration in a closed system or in an open system for the purpose of extracting energy).

**Comment by KathiSchleidt** [comment]
Very good questions, nice that somebody is clearly detailing!

To the various types of groundwater extraction (Q2&4), from the EF view it's not relevant for what it's being extracted, just provides the amounts together with relevant methodology.

To the use of EF vs PF (Q1&3), would depend on the data available for these facilities. EF would be required in either case if you're providing the amounts (Q2&4) via the EF Model. Adding PF would depend on if you have additional information on these facilities that cannot be provided by EF, but is covered by PF.

To Q5 on groundwater based subsurface energy systems - sounds like something for the RenewableAndWasteResource or RenewableAndWastePotentialCoverage types from ER (the coverage model has a GeothermalPotentialValue codelist foreseen)

**Comment by sMorrone** [comment]
Dear @HanWelmer & @KathiSchleidt ,
I fully agree that the choice of most suitable INSPIRE data theme strongly depends on the specific information available. Generally speaking, EF is to be preferred when the facilities link to observations and measurements taken.
Likewise for Q5: the data theme selection depends on whether the available data focuses on resources (in this case I would go for ER) or on energy systems (probably PF fits better).

**Comment by HanWelmer** [comment]
Dear Stefania Morrone,
Sorry for not closing the subject.

We have included Geonovum in this discussion. Together, mainly based on this discussion, we came to the following conclusions:

- we will map GUF (facilities) to PF.
- where GPD (production data) is available we will map the combination of GPD + GUF to EF.
- we will not additionally map groundwater based subsurface energy systems to ER as GPD/GUF concentrates on the facilities and water volume and it doesn't really fit the definition of geothermic energy.

All participants, thanks a lot for your contribution.
